src/api/lib/mails/transporter.js

```diff
@@ -0,0 +1,27 @@
+import { htmlToText } from 'nodemailer-html-to-text';
+import smtpTransport from 'nodemailer-smtp-transport';
+import nodemailer from 'nodemailer';
+import thunkify from 'thunkify';
+
+const transporterFactory = config => {
+    let transporter;
+
+    if (!config) throw Error('Configuration needed for transporter factory');
+
+    switch (config.service) {
+    case 'smtp':
+        transporter = nodemailer.createTransport(smtpTransport(config.auth || {}));
+        break;
+    default:  // eslint-disable-line no-case-declarations
+        const transportName = (config.transport) ? config.transport : 'console';
+        const transport = require(`./transporters/${transportName}`);
```

Comment:
Isn't it possible to do the require in config itself, as config is already javascript, no logic needed here